lib/src/impl_js/impl_js.digest.dart

```diff
@@ -14,9 +14,9 @@
 
 part of 'impl_js.dart';
 
-class _Hash implements Hash {
+class _HashImpl implements HashImpl {
```

Comment:
```suggestion
final class _HashImpl implements HashImpl {
```

People shouldn't be importing from `src/` and we shouldn't be subclassing it ourselves.

But if we make it `final` we make it clear to future us, that we have decided not to subclass `_HashImpl`.